src/RuleSet/Sets/PERCS1x0RiskySet.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\RuleSet\Sets;
+
+use PhpCsFixer\RuleSet\AbstractRuleSetDescription;
+
+/**
+ * @internal
```

Comment:
I think the intent on that one was to again discourage people from using `PER` specifically, since that should be deprecated and removed.  I may have over-interpreted it to include elsewhere.  We can remove them.

Reply:
The classes are internal in terms of PHP API, so we can freely change it's signature.

Reply:
But it does not make sense for me.. internal, but for public usage? 🤷‍♂️

Reply:
For public usage via configuration as `@Symfony` set etc., the end user does not care how is the implementation made.

Reply:
It makes sense 👌. Let's keep it.